package.json

```diff
@@ -202,7 +202,21 @@
           ]
         }
       }
-    }
+    },
+    "jsonValidation": [
+      {
+        "fileMatch": ".prettierrc",
+        "url": "http://json.schemastore.org/prettierrc"
+      }
+    ],
+    "languages": [
+      {
```

Comment:
We also support YAML in `.prettierrc` files but it doesn't make sense to have two files in one language mode...